docs/features/advanced-usage/llm-security.mdx

```diff
@@ -3,49 +3,117 @@ title: "LLM Security"
 
-## How It Works
+While Generative AI unlocks almost endless possibilities to innovate, but it's crucial to stay on top of the ever-growing number of risks and be informed and prepared accordingly.
 
-LLM Security enhances OpenAI chat completions with automated security checks:
+## How LLM Security Works
+To enable LLM security in Helicone, simply add `Helicone-LLM-Security-Enabled: true` to your request headers as follows:  
 
-- Only the last user message is checked for threats.
-- Utilizing [Prompt Armor](https://promptarmor.com), we quickly identify and block injection threats.
-- Detected threats result in an immediate block, with details sent back to you:
-  ```json
-  {
-    "success": false,
-    "error": {
-      "code": "PROMPT_THREAT_DETECTED",
-      "message": "Prompt threat detected. Your request cannot be processed.",
-      "details": "See your Helicone request page for more info."
+<CodeGroup>
+
+```bash Curl
+curl https://oai.hconeai.com/v1/completions \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer <YOUR_API_KEY>' \
+  -H 'Helicone-LLM-Security-Enabled: true' \ # add this header and set to true
+  -d '{
+    "model": "text-davinci-003",
+    "prompt": "How do I enable LLM security with helicone?",
+}'
+```
+
+```python Python
+openai.api_base = "https://oai.hconeai.com/v1"
+
+openai.Completion.create(
+    model="text-davinci-003",
+    prompt="How do I enable LLM security with helicone?",
+    headers={
+      "Helicone-LLM-Security-Enabled": "true", # add this header and set to true
     }
-  }
-  ```
-- Our checks add minimal latency, ensuring a smooth experience for compliant requests.
+)
+```
+
+```ts Node.js 
+import { Configuration, OpenAIApi } from "openai";
+const configuration = new Configuration({
+  apiKey: process.env.OPENAI_API_KEY,
+  basePath: "https://oai.hconeai.com/v1",
+  defaultHeaders: {
+    "Helicone-LLM-Security-Enabled": "true", // add this header and set to true
+  },
+});
+const openai = new OpenAIApi(configuration);
+```
+</CodeGroup>
+
+
+### Security Checks
+By enabling LLM Security, Helicone helps you enhance OpenAI chat completions with automated security checks, which includes: 
+- Checking the last user message for threats.
+- Utilizing [Prompt Armor](https://promptarmor.com/) to quickly identify and block injection threats.
+- Blocking detected threats immediately and sending details back to you in the following format: 
+    ```tsx
+    {
+      "success": false,
+      "error": {
+        "code": "PROMPT_THREAT_DETECTED",
+        "message": "Prompt threat detected. Your request cannot be processed.",
+        "details": "See your Helicone request page for more info."
+      }
+    }
+    ```
+- Adding minimal latency to ensure a smooth experience for compliant requests.
+
+**Interested in beta testing upcoming detectors?** [Schedule a call](https://cal.com/cole-gottdank/30min) with us.
```

Comment:
we should add this to all our docs

Reply:
would beta testing apply to every feature?